### PR TITLE
Adds web certs to open edX learner's guide

### PR DESCRIPTION
**Pull request by lamagnifica**

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
## [DOC-2642](https://openedx.atlassian.net/browse/DOC-2642)

Adds learner-facing information about digital certs to the OPEN edX learner's guide for Dogwood

### Date Needed

ASAP

### Reviewers

Possible roles follow. PR submitter checks the boxes after each reviewer
finishes and gives :+1:. 
- [ ] Subject matter expert: @mattdrayer 
- [ ] Subject matter expert: @nedbat 
- [x] Doc team review (sanity check/copy edit/dev edit): @srpearce @pdesjardins 
- [ ] Product review:
- [ ] Partner support: 
- [x] PM review: @marcotuts 

### Testing
- [X] Ran ./run_tests.sh without warnings or errors

### Post-review
- [x] Squash commits

File: en_us/open_edx_students/source/SFD_certificates.rst [hunk 1]
```diff
@@ -14,6 +14,63 @@ available for courses you take. For information about the certificates that
...
+Print a Web Certificate
+*****************************
+
+To print a web certificate, follow these steps.
+
+#. Open the certificate in your browser.
+
+#. In the certificate header, select **Print Certificate**.
+
+To print your certificate in the most professional looking format, note the
```
Comment (pdesjardins) [inline en_us/open_edx_students/source/SFD_certificates.rst]: I might replace "note" with "use" for simpler English.

File: en_us/open_edx_students/source/SFD_certificates.rst [hunk 2]
```diff
@@ -14,6 +14,63 @@ available for courses you take. For information about the certificates that
 your course offers, contact the course team or institution that created your
 course.
 
+.. contents::
+  :depth: 1
+  :local:
+
+==================
+Web Certificates
```
Comment (pdesjardins) [inline en_us/open_edx_students/source/SFD_certificates.rst]: Noun phase. I don't have a quick suggestion for revising.

**Comment by pdesjardins** [comment]
Looks good. :+1: I added two minor comments, non-crucial.

File: en_us/open_edx_students/source/SFD_certificates.rst [hunk 3]
```diff
@@ -14,6 +14,63 @@ available for courses you take. For information about the certificates that
...
+You can share your web certificates on Facebook, LinkedIn, and Twitter.
+
+#. Make sure that you are logged in to the social site on which you want to
+   share your certificate.
+
+#. In a different browser tab or window, open the certificate in your browser.
+
+#. Select the social icon at the top of each certificate.
+
+   A dialog for the social site you selected opens, with the certificate URL
```
Comment (srpearce) [inline en_us/open_edx_students/source/SFD_certificates.rst]: dialog box

**Comment by srpearce** [comment]
:+1: (one minor comment)

**Comment by marcotuts** [comment]
:+1:  thanks for the ping!

File: en_us/open_edx_students/source/SFD_certificates.rst [hunk 4]
```diff
@@ -14,6 +14,63 @@ available for courses you take. For information about the certificates that
...
+
+Share Your Web Certificates
+*****************************
+
+You can share your web certificates on Facebook, LinkedIn, and Twitter.
+
+#. Make sure that you are logged in to the social site on which you want to
+   share your certificate.
+
+#. In a different browser tab or window, open the certificate in your browser.
```
Comment (mattdrayer) [inline en_us/open_edx_students/source/SFD_certificates.rst]: This seems like an odd way to describe the process -- also we could maintain context by refering to the cert as 'your web certificate'

File: en_us/open_edx_students/source/SFD_certificates.rst [hunk 5]
```diff
@@ -14,6 +14,63 @@ available for courses you take. For information about the certificates that
...
+
+#. Select the social icon at the top of each certificate.
```
Comment (mattdrayer) [inline en_us/open_edx_students/source/SFD_certificates.rst]: at the top of 'your web certificate'

File: en_us/open_edx_students/source/SFD_certificates.rst [hunk 6]
```diff
@@ -14,6 +14,63 @@ available for courses you take. For information about the certificates that
...
+
+   A dialog for the social site you selected opens, with the certificate URL
+   already entered.
+
+#. Complete the steps for the social site to post your certificate.
```
Comment (mattdrayer) [inline en_us/open_edx_students/source/SFD_certificates.rst]: 'your web certificate'

File: en_us/open_edx_students/source/SFD_certificates.rst (hunk 1 above)
Comment (mattdrayer) [inline en_us/open_edx_students/source/SFD_certificates.rst]: your web certificate

File: en_us/open_edx_students/source/SFD_certificates.rst [hunk 7]
```diff
@@ -14,6 +14,63 @@ available for courses you take. For information about the certificates that
...
+
+.. _Print a Web Certificate:
+
+Print a Web Certificate
+*****************************
+
+To print a web certificate, follow these steps.
+
+#. Open the certificate in your browser.
```
Comment (mattdrayer) [inline en_us/open_edx_students/source/SFD_certificates.rst]: your web certificate

File: en_us/open_edx_students/source/SFD_certificates.rst [hunk 8]
```diff
@@ -14,6 +14,63 @@ available for courses you take. For information about the certificates that
...
+#. Select the social icon at the top of each certificate.
+
+   A dialog for the social site you selected opens, with the certificate URL
+   already entered.
+
+#. Complete the steps for the social site to post your certificate.
+
+.. _Print a Web Certificate:
+
+Print a Web Certificate
```
Comment (mattdrayer) [inline en_us/open_edx_students/source/SFD_certificates.rst:59]: Print Your Web Certificate

File: en_us/open_edx_students/source/SFD_certificates.rst (hunk 4 above)
Comment (lamagnifica) [inline en_us/open_edx_students/source/SFD_certificates.rst]: my error in repeating "in your browser" at end of sentence...

**Comment by lamagnifica** [comment]
merging this PR will fix the error that Travis is throwing

**Comment by mattdrayer** [comment]
LGTM!  :+1:

**Pull request merged**